pom.xml

```diff
@@ -1697,6 +1723,19 @@
             <groupId>org.codehaus.groovy</groupId>
             <artifactId>groovy-all</artifactId>
           </exclusion>
+          <!-- Hive 2.3.4 -->
+          <exclusion>
```

Comment:
Exclude `parquet-hadoop-bundle`, otherwise:
```java
build/sbt clean package -Phadoop-3.2 -Phive
...
[error] /home/yumwang/opensource/spark/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala:36: value JobSummaryLevel is not a member of object org.apache.parquet.hadoop.ParquetOutputFormat
[error] import org.apache.parquet.hadoop.ParquetOutputFormat.JobSummaryLevel
```

Reply:
These several exclusions would apply to both Hive 2 and Hive 1 in the build as it is now. That's probably OK; maybe they don't even exist in Hive 1. But some like this one I'm not as sure about?

Reply:
Yes. `org.apache.parquet:parquet-hadoop-bundle` don't exist in Hive 1. It shoud be `com.twitter:parquet-hadoop-bundle` in Hive 1: https://github.com/apache/hive/blob/release-1.2.1/pom.xml#L256-L260